### PR TITLE
fix: grammar

### PR DESCRIPTION
**Pull request by Rexogamer**
| Q                 | A
| ----------------- | ---
| Bug fix?          | kinda
| New feature?      | no
| Breaking changes? | no
| Deprecations?     | no
| New tests added?  | not needed
| Fixed tickets     | none
| License           | MIT

### Description

This PR fixes some grammar in the preferences.

### CODE REVIEW

File: src/renderer/prefComponents/editor/index.vue [hunk 1]
```diff
@@ -34,83 +34,83 @@
...
     ></bool>
     <bool
-      description="Hide link popup when the cursor is hover on the link."
+      description="Whether to hide the link popup when the cursor is hovering on links."
       :bool="hideLinkPopup"
       :onChange="value => onSelectChange('hideLinkPopup', value)"
     ></bool>
     <bool
-      description="Whether to automatically check related task."
+      description="Whether to automatically check any child tasks."
```
Comment (MrHeer) [inline src/renderer/prefComponents/editor/index.vue]: parent tasks maybe automatically checked.

**Comment by Rexogamer** [comment]
(The Travis fail is a 403 when trying to get rigrep on Mac)

File: src/renderer/prefComponents/editor/index.vue [hunk 2]
```diff
@@ -34,83 +34,83 @@
...
     ></cur-select>
     <bool
-      description="Try to automatically guess the file encoding when opening files."
+      description="Automatically guess the file's encoding when opening files."
       :bool="autoGuessEncoding"
       :onChange="value => onSelectChange('autoGuessEncoding', value)"
     ></bool>
     <cur-select
-      description="Whether a single trailing newline should be ensured or trailing newlines should be removed."
+      description="Whether trailing newlines should be removed."
```
Comment (fxha) [inline src/renderer/prefComponents/editor/index.vue]: I think a good improvement would be >The trailing newlines options.< or similar.

File: src/renderer/prefComponents/editor/index.vue [hunk 3]
```diff
@@ -34,83 +34,83 @@
...
     ></cur-select>
     <bool
-      description="Hide hint for quickly creating paragraphs."
+      description="Whether to hide the hint for quickly creating paragraphs."
       :bool="hideQuickInsertHint"
       :onChange="value => onSelectChange('hideQuickInsertHint', value)"
     ></bool>
     <bool
-      description="Hide link popup when the cursor is hover on the link."
+      description="Whether to hide the link popup when the cursor is hovering on links."
```
Comment (fxha) [inline src/renderer/prefComponents/editor/index.vue]: Why not just >[...] cursor is over a link<?

File: src/renderer/prefComponents/general/index.vue [hunk 4]
```diff
@@ -2,12 +2,12 @@
   <div class="pref-general">
     <h4>General</h4>
     <bool
-      description="Automatically save the content being edited."
+      description="Automatically save changes."
```
Comment (fxha) [inline src/renderer/prefComponents/general/index.vue]: I think >Automatically save document changes< would be easier to understand, what do you think @Rexogamer?

File: src/renderer/prefComponents/general/index.vue [hunk 5]
```diff
@@ -17,19 +17,19 @@
...
     <separator></separator>
     <bool
-      description="Open files in new window."
+      description="Whether to open files in a new window."
       :bool="openFilesInNewWindow"
       :onChange="value => onSelectChange('openFilesInNewWindow', value)"
     ></bool>
     <bool
-      description="Open folder via menu in a new window."
+      description="Whether to open folders via a menu in a new window."
```
Comment (fxha) [inline src/renderer/prefComponents/general/index.vue]: >Whether to open folders in a new window.< should be enough like above.

File: src/renderer/prefComponents/editor/index.vue [hunk 6]
```diff
@@ -34,83 +34,83 @@
...
-      description="The default end of line character. If you select default, the ending will be selected according to your system intelligence."
+      description="The newline character used at the end of each line. The "default" option selects your operating system's default newline character."
       :value="endOfLine"
       :options="endOfLineOptions"
       :onChange="value => onSelectChange('endOfLine', value)"
     ></cur-select>
     <separator></separator>
     <cur-select
-      description="The default file encoding."
+      description="The default encoder for files."
```
[the remark this answers is not in the document]
Comment (Rexogamer) [reply src/renderer/prefComponents/editor/index.vue]: I've changed the wording to be more similar to the original but I'm still not quite certain about it. However, I'm happy enough with it.

File: src/renderer/prefComponents/general/index.vue (hunk 4 above)
Comment (Rexogamer) [reply src/renderer/prefComponents/general/index.vue]: I agree, and I've changed it in the latest commit 👍

**Pull request merged**